content/core-features/presence-and-occupancy/topic-presence_ia-update.textile

```diff
@@ -0,0 +1,737 @@
+
+
+h3(#member-list). Presence member list
+
+h4(#realtime-member-list). Realtime API
+
+The @Presence@ object exposes a "<span lang="default">@get@</span><span lang="csharp">@Get@</span>":/api/realtime-sdk/presence#get method allowing a client to retrieve an array of all members present on the channel. The Ably client is responsible for keeping track of the presence set from the time that the channel is attached; an up to date presence set is pushed to the client following attach and the presence set is updated on each subsequent presence event. Thus "<span lang="default">@get@</span><span lang="csharp">@Get@</span>":/api/realtime-sdk/presence#get returns the already-known presence set retained in memory and does not trigger a new request to the Ably service.
+
+```[jsall]
+channel.presence.get(function(err, members) {
+  console.log('There are ' + members.length + ' members on this channel');
+  console.log('The first member has client ID: ' + members[0].clientId);
+});
+```
+
+```[ruby]
+channel.presence.get do |members|
+  puts "There are #{members.size} members on this channel"
+  puts "The first member has client ID: #{members.first.client_id}"
+end
+```
+
+```[java]
+PresenceMessage[] members = channel.presence.get();
+System.out.println("There are " + members.length + " members on this channel");
+System.out.println("The first member has client ID: " + members[0].clientId);
+```
+
+```[csharp]
+IEnumerable<PresenceMessage> presence = await channel.Presence.GetAsync();
+Console.WriteLine($"There are {presence.Count()} members on this channel");
+Console.WriteLine($"The first member has client ID: {presence.First().ClientId}");
+```
+
+```[objc]
+[channel.presence get:^(NSArray<ARTPresenceMessage *> *members, ARTErrorInfo *error) {
+    NSLog(@"There are %lu members on this channel", [members count]);
+    NSLog(@"The first member has client ID: %@", members[0].clientId);
+}];
+```
+
+```[swift]
+channel.presence.get { members, error in
+    print("There are \(members.count) members on this channel")
+    print("The first member has client ID: \(members[0].clientId)")
+}
+```
+
+h4(#rest-member-list). REST API
+
+The @Presence@ object exposes a "<span lang="default">@get@</span><span lang="csharp">@Get@</span>":#get method allowing a client to retrieve an array of all members currently present on the channel. In the REST client library this method directly queries "Ably's REST presence API":/api/rest-api#presence. No presence state is cached in the library itself, unlike in the "Realtime client library":/realtime.
+
```

Comment:
It would be nice to add some intro text to explain what the examples are doing here.